ArduSub/Sub.h

```diff
@@ -233,7 +235,6 @@ class Sub : public AP_HAL::HAL::Callbacks {
         struct {
             uint8_t unused1             : 1; // 0
             uint8_t simple_mode         : 2; // 1,2     // This is the state of simple mode : 0 = disabled ; 1 = SIMPLE ; 2 = SUPERSIMPLE
-            uint8_t pre_arm_rc_check    : 1; // 3       // true if rc input pre-arm checks have been completed successfully
```

Comment:
The only place .value is used is in a 1Hz df logging message. I am removing the indices from comments until we reach a point where all fields have been sorted to the point where they are unlikely to change in the future.